Neos.FluidAdaptor/Classes/ViewHelpers/DebugViewHelper.php

```diff
@@ -55,22 +55,31 @@ class DebugViewHelper extends AbstractViewHelper
      */
     protected $escapeOutput = false;
 
+    /**
+     * Arguments initialization
+     *
+     * @return void
+     */
+    public function initializeArguments()
+    {
+        $this->registerArgument('title', 'string', 'The title', false, null);
+        $this->registerArgument('ypeOnly', 'boolean', 'Whether only the type should be returned instead of the whole chain.', false, false);
```

Comment:
That's wrong and needs to be fixed still. 'typeOnly'

Reply:
That PR was merged in spite of this typo

Reply:
-> #1183